### ISSUE TITLE
Sacrifice Support calculation neglecting other sources of sacrificed Life like Rathpith Globe or Demon Stitcher

### ISSUE THREAD

**Issue by Vanqy9316** [PathOfBuildingCommunity/PathOfBuilding#8050]
### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### What platform are you running Path of Building on?

Windows

### Is your feature request related to a problem?

Since the TotA league and the introduction of the Sacrifice Support gem i was sold to play such a support gem based build.
With patience and a lot of dead ends during my journey to figure out such build, i eventually found the right combination of techniques available to make the build working. The main problem/solution aspect of the build was to recover the life sacrificed.
Using Rathpith Globe on a Life-stacker is a no-brainer, but the 10% sacrifice upon trigger/cast provided by the shield bugged me out - and simply testing if it affects the DPS was impossible due to its natural design.

This league im approaching to develop the version 3 of my build. After TotA and Affliction being extremely successful leagues for my build, i want to push things way further than last time.

And i finally could test how Life sacrificed from other sources than the Sacrifice Support gem itself does interfere with the added chaos damage.
The Demon Stitcher Gloves provide up to 25% Life sacrificed per cast. Testing in the Foothills of A9 brought me to the conclusion that it definitely affects the added chaos damage of the spell used with Sacrifice Support. (Boulderneck took roughly 55% of his life compared to 25% with without Demon Stitchers equipped per Hexblast casted)

So for the past two attempts building this build, my sense of having better DPS than the PoB proposes was right, so i kindly ask to implement Life sacrificed as a feature.

### Describe the solution you'd like

Global variable for:
Life sacrificed per Spell cast
Life sacrificed per Spell triggered

implementation of those two variables into Sacrifice Support:
Supported Spells Sacrifice 20% of your current Life when Cast to gain added Chaos Damage equal to (25-44)% of Sacrificed Life
added flat chaos dmg = (20% + Life sacrificed per Spell cast + Life sacrificed per Spell triggered) * (25-44)% local efficiency

### Describe alternatives you've considered

impossible

**Comment by LocalIdentity** [comment]
Just tested this interaction and it does not work as you have put above. My guess is that you didn't account for the life roll on the gloves when testing with and without them

**Comment by Vanqy9316** [comment]
my life pool is 10685 with Demon Stitchers and 10716 with my rare gloves. I did another testing with perfect agony allocated to rule out crit chance and void manipulation to make sure hexblast doesnt ignite the boss.
This time i have tested the damage on the boss in Desecrated Chambers of A10

Indeed it seems like the life sacrificed doesnt have any effect on the damage being output. i needed exactly 20 hits with either gloves on a dps wise cut down setup (no curse, 3L setup).

I have now filed a bug report in the PoE Forums. The support gem makes not clear if the added chaos damage is only based on the 20% life or not. Shall they fix their shit and give this support gem some love.

If they decide to fix this behavior, ill reach out to you again

Thanks
